# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gcc/hotfix-no-multilib.patch.musl
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gcc-12.1.0/gcc/config.gcc.vanilla	2022-07-14 12:24:47.863690831 +0200
+++ gcc-12.1.0/gcc/config.gcc	2022-07-14 12:25:37.807688840 +0200
@@ -2026,7 +2026,7 @@
 			x86_multilibs="mx32"
 			;;
 		*)
-			x86_multilibs="m64,m32"
+			x86_multilibs="m64"
 			;;
 		esac
 	fi
--- gcc-12.1.0/gcc/config/i386/t-linux64.vanilla	2022-07-14 12:37:55.609659421 +0200
+++ gcc-12.1.0/gcc/config/i386/t-linux64	2022-07-14 12:38:22.400658353 +0200
@@ -33,6 +33,6 @@
 comma=,
 MULTILIB_OPTIONS    = $(subst $(comma),/,$(TM_MULTILIB_CONFIG))
 MULTILIB_DIRNAMES   = $(patsubst m%, %, $(subst /, ,$(MULTILIB_OPTIONS)))
-MULTILIB_OSDIRNAMES = m64=../lib64$(call if_multiarch,:x86_64-linux-gnu)
+MULTILIB_OSDIRNAMES = m64=../lib$(call if_multiarch,:x86_64-linux-gnu)
 MULTILIB_OSDIRNAMES+= m32=$(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)$(call if_multiarch,:i386-linux-gnu)
 MULTILIB_OSDIRNAMES+= mx32=../libx32$(call if_multiarch,:x86_64-linux-gnux32)
